src/stores/gameList.js

```diff
@@ -36,6 +36,10 @@ export const useGameListStore = defineStore('gameList', {
       startedGame.status = GameStatus.STARTED;
       this.spectateGames.push(startedGame);
     },
+    doesGameExist(gameId) {
+      const game = this.openGames.find((game) => game.id == gameId);
+      return (game) ? true : false;
+    },
```

Comment:
I think we can one-line this

```suggestion
    doesGameExist(gameId) {
      return this.openGames.some(({id}) => id === gameId);
    },
```

Reply:
I didn't know there was a function that could do that, thanks a lot! Added your suggestion, much appreciated